tests/integration/targets/omem_smart_fabric_info/tests/info-12341.yaml

```diff
@@ -0,0 +1,88 @@
+# Dell OpenManage Ansible modules
+# Copyright (C) 2023-2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an OME host, OME username and OME password
+  ansible.builtin.fail:
+    msg: 'Please define the following variables: hostname, username
+         and password.'
+  when: hostname is not defined or username is not defined or password
+         is not defined
+
+- name: Get all fabric information
+  block:
+    - ansible.builtin.include_tasks: _fabric_info_uri_helper.yml
```

Comment:
Add task name for this and following task.
Applicable for tasks in all playbooks,